com.unity.cinemachine/Samples~/3D Samples/ClearShot.unity

```diff
@@ -0,0 +1,1653 @@
+%YAML 1.1
```

Comment:
![image](https://user-images.githubusercontent.com/6424658/205764190-622354f2-0d19-4d1c-aafe-57222817d53c.png)

Reply:
I don't like the help object being hidden as a child of the camera.  It should be a top-level object, for clarity.  Same comment applies to all the sample scenes.

Reply:
The character should turn around and walk back, or warp back to the starting point when it reaches the end of its track.  Maybe you can use the WrapAround script to make that happen without adding new custom scripts.

Reply:
I had a custom script to turn around the player, but did not commit it (it was in my local assets). Pushed it. I did not like the wrap around script it is too abrupt in this case in my opinion.